CODEOWNERS

```diff
@@ -975,6 +975,7 @@
 /tests/bluetooth/bsim/nrf_auraconfig/*.rst @nrfconnect/ncs-audio-doc
 
 /tests/samples/bluetooth/io_adapter/      @rafpelc @jema-nordic
+/tests/samples/bluetooth/samples_test_app/ @rafpelc
```

Comment:
Please add an entry for the RST 2 lines above like the other entries in that section.

Reply:
Added separate entry for RST file